project/redis-master.sls

```diff
@@ -0,0 +1,17 @@
+# Install Redis, listening on all interfaces.
+# This does not open the firewall for Redis, so do that elsewhere
```

Comment:
What's the point of listening on all interfaces if you can't access it outside of localhost?